pkg/network/cache/client.go

```diff
@@ -27,6 +28,7 @@ type (
 		ResponseCallback func(client.ResponseMetaInfo) error
 		AllowExternal    bool
 		Buffers          *sync.Pool
+		Logger           *zap.Logger
```

Comment:
`NewSDKClientCache` must fallback to no-op log if opt is unset

Reply:
not sure i agree cause it is kinda required thing in the current context and this PR means we _want to log it_ but ok

Reply:
but i told about `NewSDKClientCache`... ok, nvm